Automattic/wp-calypso
# Help Center: fix icons and whats new

#### Changes proposed in this Pull Request

Fixed the `More resources` section to use proper icons, and introduced `What's New` icon. Its behaviour should be the same as clicking on `What's new` in the menu:
![image](https://user-images.githubusercontent.com/52076348/165804087-30667565-2ab0-48c3-99ee-15e6e2f6ad21.png)

To know if it has to show to dot or not, a new element has been introduced in the help-center store, and 2 new APIs in What's new.

First thing now we check in local storage, to see if we have saved if the user has seen the Whats New modal or not. If the value isn't in the local storage yet (we added `persist: true` to the help-center store), we call the API to fetch the value from `user_meta`. We avoid a lot of network calls using the store persistence.

#### Testing instructions



* Checkout the branch
* `yarn dev --sync` from `apps/editing-toolkit`
* Make sure to have you public API sandboxed
* Verify that `More resources` is aligned to [design](pbAok1-2D2-p2)
![image](https://user-images.githubusercontent.com/52076348/165803676-9634b257-e82d-405d-8ee1-4474779ed2c9.png)
* What's new should have the dot next to it, also the help center icon should have the little dot
* Verify that `What's new` opens the modal
* Verify that works in `http://calypso.localhost:3000/checkout/yoursite.wordpress.com` as well
* Verify that the dot next to `What's new` is gone, as well as the circle next to the help center icon
